src/event.rs

```diff
@@ -338,6 +337,11 @@ impl<N: Notify> Processor<N> {
                         }
 
                         processor.on_focus_change(is_focused);
+                    },
+                    DroppedFile(path) => {
+                        use input::ActionContext;
+                        let path: String = path.to_string_lossy().into();
+                        processor.ctx.write_to_pty(path.into_bytes());
                     }
```

Comment:
Note to future self: Write up a PR allowing the user to configure this behavior through `alacritty.yaml` like any other input.